test/framework/tutorials/tutorial_06_wdd.cc

```diff
@@ -16,6 +16,8 @@
 
 #include "console/chi_console.h"
 
+#include <iomanip>
```

Comment:
I have no idea why this showed up suddenly. Guess I was including it somewhere and now I am not.